docs/nx/playwright/init.md

```diff
@@ -48,7 +37,10 @@ Any existing properties in the _playwright.config.base.ts_ file will have their
 │   │   ├── src
 │   │   │   ├── example.spec.ts #Example tests using playwright
```

Comment:
`app.spec.ts` is also generated here alongside example.spec.ts
`.eslintrc.json` is also generated at e2e root.